parquet-hadoop/src/main/java/org/apache/parquet/column/values/factory/ConfigurableFactory.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.parquet.column.values.factory;
+
+import org.apache.hadoop.conf.Configuration;
+
+/**
+ * Used with {@link ValuesWriterFactory} to indicate a factory that has Hadoop config included.
+ */
+public interface ConfigurableFactory {
```

Comment:
Ah nice. This is exactly what I'm doing. Just that they truncated the method names :-). Replaced my interface.